### ISSUE TITLE
Testing against various pieces of code

### ISSUE THREAD

**Issue by alystair** [workman-layout/Workman#56]
As a programmer I rather see tests results from of pieces of programming (JS, C++, PHP etc) instead of complete novels.

Please add some analytics where you show Workman's comparison for code bases instead of novels. Curious to see if the results differ.

**Comment by fhaust** [comment]
Yep ... came here to suggest testing on some open source projects ... Linux Kernel, Firefox, Chromium etc.

**Comment by ctolkien** [comment]
>Linux Kernel, Firefox, Chromium etc.

With most editors, you're not actually typing out those letters however - your editor/IDE is going to do a lot of lifting for you.  You need to log the keystrokes you actually type. Does anyone know a good tool to do this?

I would :+1: however that some type of code is going to be more realistic than Alice in Wonderland.

**Comment by CircleOnCircles** [comment]
> > Linux Kernel, Firefox, Chromium etc.
> 
> With most editors, you're not actually typing out those letters however - your editor/IDE is going to do a lot of lifting for you. You need to log the keystrokes you actually type. Does anyone know a good tool to do this?
> 
> I would 👍 however that some type of code is going to be more realistic than Alice in Wonderland.

agreed

On a surface it might seems taking the source code and run a typing analysis over them is a good idea, actual typing in IDE involves a lot of autocomplete and autosuggest. anyway, it is better than nothing to have such a naive benchmark.

**Comment by CircleOnCircles** [comment]
Here is the test I create using python code of a file on the most popular repo.
http://patorjk.com/keyboard-layout-analyzer/#/load/bV0wSC19

After a few test  on different file. workman-p is not came up at the top.
